docs/UserGuide.adoc

```diff
@@ -1035,33 +1099,67 @@ General notes are found at the bottom of the rightmost column on the offline cop
-* *Add*: `*add* u/URL n/NAME [r/REMARK] [f/FOLDER] [t/TAG]…​`
-* *List*: `*list*`
-* *Edit*: `*edit* INDEX [u/URL] [n/NAME] [r/REMARK] [f/FOLDER] [t/TAG]...​`
-* *Find*: `*find* [KEYWORD]... [t/TAG]... [f/FOLDER]...`
-* *Delete*: `*delete* INDEX`
-* *Goto*: `*goto* INDEX`
-* *Favorite*: `*favorite|fav* INDEX`
-* *Clear*: `*clear*`
-* *Undo*: `*undo* [STEP=1]`
-* *Redo*: `*redo* [STEP=1]`
-* *Exit*: `*exit*`
-* *Export*: `*export* FILENAME`
-* *Import*: `*import* FILENAME`
-* *Autotag (add)*: `*autotag* TAG_NAME [n/NAME_KEYWORD]... [u/URL_KEYWORD]... [f/FOLDER]...
+.Summary of command formats
+[%autowidth]
+|=====
+|*Category* |*Commands*
+.8+|General
+| Help: `*help*`
+| Tab: `*tab* INDEX`
+| Undo: `*undo* [STEP=1]`
+| Redo: `*redo* [STEP=1]`
+| Export: `*export* FILENAME`
+| Import: `*import* FILENAME`
+| Clear: `*clear*`
+| Exit: `*exit*`
+
+.7+|Bookmarks
+| List: `*list*`
+| Find: `*find* [KEYWORD]... [t/TAG]... [f/FOLDER]...`
+| Goto: `*goto* INDEX`
+| Add bookmark: `*add* u/URL n/NAME [r/REMARK] [f/FOLDER] [t/TAG]…​`
+| Edit bookmark: `*edit* INDEX [u/URL] [n/NAME] [r/REMARK] [f/FOLDER] [t/TAG]...​`
+| Delete bookmark: `*delete* INDEX`
+| Favorite bookmark: `*favorite\|fav* INDEX`
+
+.3+|Folders
+| Expand view: `*expand* [LEVEL=1]`
+| Collapse view: `*collapse* [LEVEL=1]`
+| Add folder: `*folder* FOLDER_NAME [p/PARENT_FOLDER=ROOT]`
+
+.2+|Autotags
+| Add autotag: `*autotag* TAG_NAME [n/NAME_KEYWORD]... [u/URL_KEYWORD]... [f/FOLDER]...
 [nn/NOT_NAME_KEYWORD]... [nu/NOT_URL_KEYWORD]... [nf/NOT_FOLDER]...`
-* *Autotag (delete)*: `*autotag-delete* TAG_NAME`
-* *Cache (add)*: `*cache* INDEX`
-* *Cache (delete)*: `*cache-delete* INDEX`
-* *Offline*: `*offline* INDEX`
-* *Annotate (add)*: `*annotate* INDEX p/P_NUM [n/NOTES] [h/HIGHLIGHT_COLOUR=yellow]`
-* *Annotate (edit)*: `*annotate-edit* INDEX p/P_NUM [to/NEW_P_NUM] [n/NOTES] [h/HIGHLIGHT_COLOUR]`
-* *Annotate (delete)*: `*annotate-delete* INDEX [p/P_NUM] [n/KEEP_NOTES=false] [h/KEEP_HIGHLIGHT=false]`
-* *Reminder (add)*: `*reminder* INDEX t/TIME_DATE [n/NOTE=Open]`
-* *Reminder (edit)*: `*reminder-edit* INDEX [t/TIME_DATE] [n/NOTE]`
-* *Reminder (delete)*: `*reminder-delete* INDEX`
-* *Folder (add)*: `*folder* FOLDER_NAME [p/PARENT_FOLDER=ROOT]`
-* *Expand*: `*expand* [LEVEL=1]`
-* *Collapse*: `*collapse* [LEVEL=1]`
-* *Tab*: `*tab* INDEX`
+| Delete autotag: `*autotag-delete* TAG_NAME`
+
+.6+|Offline Copies
+| Add cache: `*cache* INDEX`
+| Delete cache: `*cache-delete* INDEX`
+| Offline view: `*offline* INDEX`
+| Add annotation: `*annotate* INDEX p/P_NUM [n/NOTES] [h/HIGHLIGHT_COLOUR=yellow]`
+| Edit annotation: `*annotate-edit* INDEX p/P_NUM [to/NEW_P_NUM] [n/NOTES] [h/HIGHLIGHT_COLOUR]`
+| Delete annotation: `*annotate-delete* INDEX [p/P_NUM] [n/KEEP_NOTES=false] [h/KEEP_HIGHLIGHT=false]`
+
+.3+|Reminders
+| Add reminder: `*reminder* INDEX t/TIME_DATE [n/NOTE=Open]`
+| Edit reminder: `*reminder-edit* INDEX [t/TIME_DATE] [n/NOTE]`
+| Delete reminder: `*reminder-delete* INDEX`
```

Comment:
```suggestion
| Delete reminder: `*reminder-delete* INDEX`
| Goto reminder: `*reminder-goto* INDEX`
```

Reply:
I added it as `*goto-reminder*` based on the current implementation!

Reply:
I'll do it.